src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  margin: 0;
+}
```

Comment:
don't reset your margins with * selector, it's bad practice, you only need to reset margin for body in this task